lib/remoteproc/remoteproc.c

```diff
@@ -900,6 +900,16 @@ static int remoteproc_virtio_notify(void *priv, uint32_t id)
 	return 0;
 }
 
+static int remoteproc_wait_notified(void *priv, uint32_t id)
+{
+	struct remoteproc *rproc = priv;
+
+	if (rproc->ops->wait_notified)
+		return rproc->ops->wait_notified(rproc, id);
+
+	return -RPROC_EXIO;
```

Comment:
I believe EXIO means IO error. More appropriate error code would be RPROC_EOPNOTSUPP. It says operation not supported. Please rename this error code.

Reply:
Thanks, Done